### PR TITLE
Spanish Terminology - Added some terms, alphabetical order, and some notes for discussion

### PR DESCRIPTION
**Pull request by lh1008**
Hi,

I reordered everything alphabetically, added some terms, and left some comments for open discussions. The words were added with the help of the review for [monero-GUI-guide PR #55](https://github.com/monero-ecosystem/monero-GUI-guide/pull/55#issuecomment-427814799).

Thank you @erciccione and @febrezo for your work and effort. :)

### CODE REVIEW

File: terminology-guides/spanish-terminology.md [hunk 1]
```diff
@@ -4,39 +4,50 @@
...
 | Mine | Minar | |
 | Miner | Minero o Minador| |
+| Mining | Minería | |
 | Mnemonic seed | Semilla mnemónica | Mnemónica,co [RAE](http://dle.rae.es/?id=PRbUjYS) "Mnemo~ ⇒ Mnemósine, diosa griega de la memoria + técnica, que signifícan, técnicas para ayudar la memoria o, sencillamente, técnicas de memorización." Fuente [Mnemotecnia](https://www.mnemotecnia.es/inicio). |
 | Open-source | Código abierto | |
 | Paper wallet | Monedero en papel | |
 | Pool | Pool | Se queda "pool" y no "piscina", ya que pool hace referencia a esfuerzo o valor compartido |
+| Pool Mining | Minería en pool | |
+| Public view key | Clave Pública de visualización | |
+| Public spend key | Clave de gasto público / Clave pública de gasto | Open discussion |
```
Comment (BlackLotus64) [inline terminology-guides/spanish-terminology.md]: We should use similar terms like the one at line 27. So I'm up to the second choice.

File: terminology-guides/spanish-terminology.md [hunk 2]
```diff
@@ -4,39 +4,50 @@
...
 | Receiver | Receptor | |
+| Remote Node Port | Puerto de Nodo Remoto | |
+| Rescan wallet balance | Reescanear el balance del monedero / Volver a escanear el saldo del monedero | Open to discussion |
 | Ring signature | Firma de círculo | Se mantiene como firma de círculo por su uso en wikipedia |
+| Run | Ejecutar, Correr, Iniciar | Consistencia en términos ya utilizados y generalizados en diferentes Sistemas Operativos. [Término "Run" en Microsoft](https://www.microsoft.com/en-us/language/Search?&searchTerm=run&langID=Spanish&Source=true&productid=All) |
+| Secret spend key | Clave secreta de gasto | |
+| Secret view key | Clave secreta de visualización | |
```
Comment (BlackLotus64) [inline terminology-guides/spanish-terminology.md]: The same term is already being used in another word, in this case it is **view key**, the *secret* (or *public*, *private*) part it's more easy and not part of the crypto-technicalities. What I mean is that we can just leave Spend Key, View Key, Private Key as the listed words for the terminology, what do you think? :)

Comment (lh1008) [reply terminology-guides/spanish-terminology.md]: I understand, :+1:

**Comment by lh1008** [comment]
Done @BlackLotus64. Waiting for approval.

**Comment by lh1008** [comment]
Thank you @erciccione :)

**Pull request merged**